packages/charts/react-charts-preview/library/src/components/DonutChart/Pie/Pie.tsx

```diff
@@ -25,7 +25,6 @@ export const Pie: React.FunctionComponent<PieProps> = React.forwardRef<HTMLDivEl
     const classes = usePieStyles_unstable(props);
     const pieForFocusRing = d3Pie()
       .sort(null)
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
why remove this line

Reply:
was showing as an unused eslint directive